.travis.yml

```diff
@@ -153,10 +153,10 @@ jobs:
     - PLATFORM_PUBLIC_PATH=https://marketplace-staging.streamr.com
     - GOOGLE_ANALYTICS_ID=55331565-5
     - STORYBOOK_BASE_PATH=storybook
-    - MARKETPLACE_CONTRACT_ADDRESS=0x7D9d8FE2275a196341DAFf0F277a7B851329590e
+    - MARKETPLACE_CONTRACT_ADDRESS=0x438229f341AEa22f18E69D98c2bE5997bc62d578
```

Comment:
The new contract (with `redeploy` parameter in `updateProduct`) was not deployed to staging before, this is the correct address and eth watcher has been updated to watch this in staging.